Lib/gftools/builder/__init__.py

```diff
@@ -37,6 +37,7 @@ def edge_with_operation(node, operation):
 class GFBuilder:
     config: dict
     recipe: Recipe
+    fontc_args: FontcArgs
```

Comment:
do you actually use this at all later? One of the ideas of the original design was that this could be a transient thing that was just used to set some config flags and then could be thrown away, and the rationale for that was that I wanted to avoid polluting the project with stuff that I would anticipate wanting to rip out later.

If there's a reason to keep it sure, but otherwise it's probably not worth?

Reply:
Yes, see `glyphs_to_ufo` below - if we're using fontc, then UFOs should be stored in package structure, not JSON structure.